shapedetector.py

```diff
@@ -66,7 +84,14 @@ def detect_shapes():
         M = cv2.moments(c)
 
         # ignore contours that are too small to be species
-        if c.shape[0] > 2 and area/(resized.shape[0]*resized.shape[1]) > 0.002:
+        UPPERCONTOURTHRESH = 0.25
+        LOWERCONTOURTHRESH = 0.002
+
+        if c.shape[0] > 2 and\
+        area/(resized.shape[0]*resized.shape[1]) > LOWERCONTOURTHRESH and\
+        area/(resized.shape[0]*resized.shape[1]) < UPPERCONTOURTHRESH:
+            # find centroid of contour
+            # if M['m00'] == 0 shape = line
```

Comment:
why is this code commented out? should it still be here? if not you should remove it